src/Components/RegisterForm.php

```diff
@@ -51,4 +53,21 @@ public function render()
             'invitation' => $this->invitationId ? Models::invitation()::findByUuid($this->invitationId) : null,
         ]);
     }
+
+    public function canSubmit(): bool
+    {
+        $requiredProperties = ['name', 'username', 'email', 'password', 'password_confirmation', 'terms'];
+
+        if ($this->invitationId) {
+            array_push($requiredProperties, 'code');
+        }
```

Comment:
`invitation` !== `code`

code is used for MSQ invite codes. Invitation is for inviting users to your team through the UI (in addition to needing a sign up code)